server/server.go

```diff
@@ -46,9 +48,10 @@ type Config struct {
 	GRPCStreamMiddleware []grpc.StreamServerInterceptor `yaml:"-"`
 	HTTPMiddleware       []middleware.Interface         `yaml:"-"`
 
-	GPRCServerMaxRecvMsgSize       int  `yaml:"grpc_server_max_recv_msg_size"`
-	GRPCServerMaxSendMsgSize       int  `yaml:"grpc_server_max_send_msg_size"`
-	GPRCServerMaxConcurrentStreams uint `yaml:"grpc_server_max_concurrent_streams"`
+	GPRCServerMaxRecvMsgSize       int                        `yaml:"grpc_server_max_recv_msg_size"`
+	GRPCServerMaxSendMsgSize       int                        `yaml:"grpc_server_max_send_msg_size"`
+	GPRCServerMaxConcurrentStreams uint                       `yaml:"grpc_server_max_concurrent_streams"`
+	GRPCServerKeepAliveConfigs     keepalive.ServerParameters `yaml:"-"`
```

Comment:
Ideally we should allow to configure all parameters from YAML as well. What do you think to move here the 5 params we configure in `GRPCServerKeepAliveConfigs` and then when we call `grpc.KeepaliveParams()` we do create a `keepalive.ServerParameters{}` on the fly using the 5 params set here?

Reply:
That sounds good to me since we can't add struct tags to the underlying struct.